study-builder/studies/mpc/patches/consent-version2.conf

```diff
@@ -0,0 +1,345 @@
+                              the study directors.
+                              </li>
+                            </ul>
+                          </li>
+                        </ol>
+  """,
+
+  "s3_additional_agree_item": """
+              <ul class="PageContent-ul">
+                <li class="PageContent-text topMarginMedium">
+                  You can request my medical records from my physicians and the hospitals and other places where I received
+                  and/or continue to receive my treatment and link results of the gene tests you perform on my saliva and, if I elect on
+                  this form, blood and tissue samples with my medical information from my medical records.
+                </li>
+                <li class="PageContent-text topMarginMedium">
+                  You can analyze a saliva sample that I will send you, link the results to my medical information and other
+                  specimens, and store the specimen to use it for future research.
+                </li>
+                <li class="PageContent-text topMarginMedium">
+                  You can perform (or collaborate with others to perform) gene tests on the blood and saliva samples that I
+                  will send you and store the samples until this research study is complete.
+                </li>
+                <li class="PageContent-text topMarginMedium">
+                  You can use the results of the gene tests and my medical information for future research studies, including
+                  studies that have not yet been designed, studies for diseases other than cancer, and/or studies that may be for
+                  commercial purposes.
+                </li>
+                <li class="PageContent-text topMarginMedium">
+                  You can share the results of the gene tests and my medical information with established public
+                  databases (e.g., the NIH, cBioPortal, Tumor Portal, The Exome Aggregation Consortium
+                  (ExAC)/Genome Aggregation Database (gnomAD))
+                   and with other qualified researchers in a manner that does not include my name, social security number, or any other
+                  information that could be used to readily identify me, to be used by other qualified researchers to perform future
+                  research studies, including studies that have not yet been designed, studies for diseases other than cancer, and
+                  studies that may be for commercial purposes.
+                </li>
+              </ul>
+  """
+
+  "releasePdfName": "mpcproject-release",
+
+  "consentPdfV2": {
+    "name": "mpcproject-consent",
+    "filename": "mpcproject-consent",
+    "displayName": "MPC consent pdf",
+    "versions": [
+      {
+        "tag": "v2",
+        "start": "2019-05-15T17:00:00Z",
+        "end": null,
+        "sources": [
+          {
+            "type": "EMAIL"
```

Comment:
It doesn't look like the `consent_v2.pdf` file needs to substitute in participant's email?